specification/workloads/resource-manager/Microsoft.Workloads/monitors/readme.az.md

```diff
@@ -0,0 +1,16 @@
+## AZ
+
+These settings apply only when `--az` is specified on the command line.
+
+For new Resource Provider. It is highly recommended to onboard Azure CLI extensions. There's no differences in terms of customer usage. 
+
+``` yaml $(az) && $(target-mode) != 'core'
+az:
+    extensions: monitors
```

Comment:
>monitors [](http://example.com/codeflow?start=16&length=8)

it might conflict with Azure Monitor. 
Use "sapmonitors" as the extensions name.

Reply:
done